src/ecr_check_repos_for_terraform/check.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
We generally don't use this approach... it's preferred to manage your python env outside of the code itself and then use the `python` binary you have configured to execute the script.

Reply:
[works for me](https://github.com/artsy/opstools/commit/4998ee53261ed277c40d7e713822ea8812b239ba).